app/src/components/BaseTitle.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="px-4 md:px-12 pt-16 pb-8 border-b border-grey-100">
+    <div class="block md:flex gap-x-8">
+      <div class="flex flex-wrap content-center gap-x-8">
+        <div class="cursor-pointer uppercase font-medium">
+          {{ title }}
+          <span v-if="counter" class="ml-2 text-grey-400">({{ counter }})</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'BaseTitle',
```

Comment:
I think this needs a more clear name, or some comments, as I'm unsure what this component is for. From the name it sounds like this should be part of `BaseHeader`, but from the usage that is not the case

Reply:
So I think this is a section heading -- should we go for something like that? `SectionHeading` / `SectionHeader`?

Reply:
Either of those sounds good to me!

Reply:
I've gone with `SectionHeader` 🙌